src/js/hooks/cycleCount/useToCountTab.jsx

```diff
@@ -0,0 +1,276 @@
+
+const useToCountTab = ({ filterParams }) => {
+  const columnHelper = createColumnHelper();
+  const translate = useTranslate();
+  const spinner = useSpinner();
+  const [pageSize, setPageSize] = useState(5);
+  const [offset, setOffset] = useState(0);
+
+  const { currentLocale, currentLocation } = useSelector((state) => ({
+    currentLocale: state.session.activeLanguage,
+    currentLocation: state.session.currentLocation,
+  }));
+
+  const {
+    dateLastCount,
+    categories,
+    internalLocations,
+    tags,
+    catalogs,
+    negativeQuantity,
+    searchTerm,
+  } = filterParams;
+
+  const getParams = ({
+    sortingParams,
+  }) => _.omitBy({
+    status: CycleCountStatus.CREATED,
+    offset: `${offset}`,
+    max: `${pageSize}`,
+    ...sortingParams,
+    ...filterParams,
+    searchTerm,
+    facility: currentLocation?.id,
+    dateLastCount,
+    categories: categories?.map?.(({ id }) => id),
+    internalLocations: internalLocations?.map?.(({ name }) => name),
+    tags: tags?.map?.(({ id }) => id),
+    catalogs: catalogs?.map?.(({ id }) => id),
+    abcClass: [],
+    negativeQuantity,
+  }, (val) => {
+    if (typeof val === 'boolean') {
+      return !val;
+    }
+    return _.isEmpty(val);
+  });
+
+  const {
+    sortableProps,
+    sort,
+    order,
+  } = useTableSorting();
+
+  const {
+    selectRow,
+    isChecked,
+    selectHeaderCheckbox,
+    selectedCheckboxesAmount,
+    headerCheckboxProps,
+  } = useTableCheckboxes();
+
+  const {
+    tableData,
+    loading,
+  } = useTableDataV2({
+    url: CYCLE_COUNT_CANDIDATES(currentLocation?.id),
+    errorMessageId: 'react.cycleCount.table.errorMessage.label',
+    defaultErrorMessage: 'Unable to fetch products',
+    shouldFetch: filterParams.tab,
+    getParams,
+    pageSize,
+    offset,
+    sort,
+    order,
+    searchTerm,
+    filterParams,
+  });
+
+  const getProductIds = () => tableData.data.map((row) => row.product.id);
+
+  const checkboxesColumn = columnHelper.accessor('selected', {
+    header: () => (
+      <TableHeaderCell>
+        <Checkbox
+          noWrapper
+          {...headerCheckboxProps}
+          onClick={selectHeaderCheckbox(getProductIds)}
+        />
+      </TableHeaderCell>
+    ),
+    cell: ({ row }) => (
+      <TableCell className="rt-td">
+        <Checkbox
+          noWrapper
+          onChange={selectRow(row.original.product.id)}
+          value={isChecked(row.original.product.id)}
+        />
+      </TableCell>
+    ),
+    meta: {
+      getCellContext: () => ({
+        className: 'checkbox-column',
+      }),
+    },
+  });
+
+  const columns = useMemo(() => [
+    columnHelper.accessor('status', {
+      header: () => (
+        <TableHeaderCell>
+          {translate('react.cycleCount.table.status.label', 'Status')}
+        </TableHeaderCell>
+      ),
+      cell: () => (
+        <TableCell className="rt-td">
+          <StatusIndicator
+            variant="danger"
```

Comment:
let's leave a TODO to remember to change it when the statuses have the variant implemented.